provider/docker/docker.go

```diff
@@ -45,6 +45,7 @@ type Provider struct {
 	ExposedByDefault      bool             `description:"Expose containers by default" export:"true"`
 	UseBindPortIP         bool             `description:"Use the ip address from the bound port, rather than from the inner network" export:"true"`
 	SwarmMode             bool             `description:"Use Docker on Swarm Mode" export:"true"`
+	DefaultNetwork        string           `description:"Default Docker network used" export:"true"`
```

Comment:
To be homogeneous with `domain`, we need to keep `Network`.

Reply:
:grin:

Reply:
Are you both ok with Network ? :)